frontend/src/helpers/dependly-component-display.tsx

```diff
@@ -0,0 +1,29 @@
+import { ErrorIndicator, LoadingIndicator } from "assets/styles/theme";
+
+interface IDependlyComponentDisplay {
+    isLoading: boolean;
+    error: unknown;
+    data: any;
```

Comment:
Don't use any if you can't proccess data of all type.

Reply:
I will refactor it.